content/en/docs/setup/upgrade/cni-helm-upgrade/index.md

```diff
@@ -1,121 +0,0 @@
----
-title: Upgrade using Helm
-description: Upgrade the Istio control plane, and optionally, the CNI plug-in using Helm.
-weight: 30
-aliases:
-    - /docs/setup/kubernetes/upgrade/steps/
-    - /docs/setup/upgrade/steps
-keywords: [kubernetes,upgrading]
----
-
-Follow this guide to upgrade the Istio control plane and sidecar proxies of an
-existing Istio deployment that was previously installed using Helm. The upgrade
-process may install new binaries and may change configuration and API schemas.
-The upgrade process may result in service downtime. To minimize downtime,
-please ensure your Istio control plane components and your applications are
-highly available with multiple replicas.
-
-{{< warning >}}
-Be sure to check out the upgrade notes
-for a concise list of things you should know before upgrading your deployment to Istio {{< istio_version >}}.
-{{< /warning >}}
-
-{{< tip >}}
-Istio does **NOT** support skip level upgrades. Only upgrades from {{< istio_previous_version >}} to {{< istio_version >}}
-are supported. If you are on an older version, please upgrade to {{< istio_previous_version >}} first.
-{{< /tip >}}
-
-## Upgrade steps
-
-[Download the new Istio release](/docs/setup/getting-started/#download)
-and change directory to the new release directory.
-
-### Istio CNI upgrade
```

Comment:
Should this part be deleted. Is the CNI plug-in also covered by the new API?

Reply:
https://preliminary.istio.io/docs/setup/install/istioctl/#identify-an-istio-component lists`cni` as a component so `istioctl` can be used to install and upgrade via the two pages.

This page only has `helm` related instructions.

Also, this page, I believe, has been missing from preliminary.istio.io for a while. I believe it's not rendered do to the `Content directory "/work/content/en/docs/setup/upgrade" have both index.* and _index.* files, pick one.` warning. I was trying to see about a fix for that as well.

Reply:
This PR will fix the warning, because it's removing the cni_helm_upgrade